nav2_bringup/launch/cloned_multi_tb3_simulation_launch.py

```diff
@@ -0,0 +1,173 @@
+# Copyright (c) 2023 LG Electronics.
```

Comment:
Add a sentence or two in the readme about these 2 launch files about their uses and differences (e.g. one clones a single robot N times - based on commandline, the other allows for N unique robot instances - set in the launch file itself) with an example `ros2 launch ... ` of each to show an example usage.

Reply:
I'd like to add in `nav2_bringup/readme.md`.
Let's discuss after edit it.

Reply:
Yeah, the package readme, that's what I meant :smile:

Reply:
Updated.

Please check with latest commit